setup.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/bash -ex
-# The -e option would make our script exit with an error if any command
-# fails while the -x option makes verbosely it output what it does
-
-# Install Pipenv, the -n option makes sudo fail instead of asking for a
-# password if we don't have sufficient privileges to run it
-sudo -n dnf install -y pipenv
-
-cd /vagrant
-# Install dependencies with Pipenv
-pipenv sync --dev
-
-# Run database migrations
-pipenv run python manage.py migrate
-
-# run our app. Nohup and "&" are used to let the setup script finish
-# while our app stays up. The app logs will be collected in nohup.out
-setsid pipenv run python manage.py runserver 0.0.0.0:8000 > runserver.log 2>&1 &
```

Comment:
I tried to delete my branch's forked setup.sh because it is redundant, I am not sure if it deletes the original setup.sh